src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -346,6 +351,12 @@ export default class DefaultAudioVideoController
       this._reconnectController.startedConnectionAttempt(true);
     }
 
+    // audio connection is needed to trigger attendee presence event
+    // if `attendeePresenceTimeoutMs` is greater than 0,
+    // the action wait for the presence of an attendee through real time controller
```

Comment:
Should we re-word these comments? Could not get the last sentence here.

Reply:
reworded.

Reply:
I am a bit hesitant to bloat up AudioVideoControllerState by introducing another state, as the size of it is becoming larger and larger.